Index: app/src/main/java/com/silogood/s_permissions/Permissions_in.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/silogood/s_permissions/Permissions_in.java	(date 1450265113000)
+++ app/src/main/java/com/silogood/s_permissions/Permissions_in.java	(revision )
@@ -1,0 +1,0 @@
